docs/RationaleHW.md

```diff
@@ -162,6 +178,14 @@ FIRRTL dialect `InstanceGraph` type, etc.
 
 ## Parameterized Modules
 
+_MW: General question about parameters in modules... we seem to be ignoring
+the concept of a `localparam`, which is more of a #define than an actual parameter.
+Yet, this concept is still very useful from the "perceived readability of the generated code".
+Can we clarify the rationale here -- is this a "we'll get to localparam/`define later" or
+"these things aren't useful because ____"?
+NB: I kept reading and then there was a quick mention of localparam at the end. Might be worth
+calling out up front._
```

Comment:
Sounds good, we support localparam (in the sv dialect), I'll clarify that in the intro.